client/beefy/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "beefy-gadget"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+
+[dependencies]
+fnv = "1.0.6"
+futures = "0.3"
+log = "0.4"
+parking_lot = "0.11"
+thiserror = "1.0"
+wasm-timer = "0.2.5"
+
+codec = { version = "2.2.0", package = "parity-scale-codec", features = ["derive"] }
+prometheus = { package = "substrate-prometheus-endpoint", git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
I guess we can reference that directly via `path`?

Reply:
Damn, misses this one. Fixed.